tests/timeseries_util.erl

```diff
@@ -41,101 +59,91 @@
 -define(MAXFLOAT,            math:pow(2, 63)).
 -define(MULTIPLECLUSTERSIZE, 3).
 
-confirm_create(ClusterType, DDL, Expected) ->
+-spec(confirm_create(atom(), string()) -> {ok, string()} | term()).
+confirm_create(ClusterType, DDL) ->
```

Comment:
I agree with the intention of this set of changes, but I was already unhappy about `confirm` functions that didn't actually confirm the results. These all should really be renamed now.